LiskHQ/lisk-sdk
# Update commander bootstrap to support new commands - Closes #7319

### What was the problem?

This PR resolves #7319 

### How was it solved?

- Update project/module/command/plugin generation commands
- Update genesis block creation command
- Update bootstrap application to include new commands
- Updated dpos-mainchain example

### How was it tested?
1. insert below into `commander/src/bootstrapping/templates/lisk-template-ts/generators/init_generator.ts:L94`. It needs this since commander and sdk are not published
```
this.spawnCommandSync('yarn', ['link', 'lisk-sdk'], {
	cwd: this.destinationRoot(),
});
this.spawnCommandSync('yarn', ['link', 'lisk-commander'], {
	cwd: this.destinationRoot(),
});
```
2. in `sdk` folder, `yarn link`
3. in commander,
```
./bin/run init ~/Desktop/temp
```
4. check `temp` project is buildable and all the files are present
5. `/path/to/commander/bin/run generate:module mod`
6. `/path/to/commander/bin/run generate:command mod com`
